src/colmap/scene/image.h

```diff
@@ -90,6 +87,11 @@ class Image {
   // World to camera pose.
   inline const Rigid3d& CamFromWorld() const;
   inline Rigid3d& CamFromWorld();
+  inline std::optional<Rigid3d>& MaybeCamFromWorld();
```

Comment:
Is this method actually useful for something?

Reply:
Yes, when copying an existing image into a new one without having to check whether it has a pose.